pagefaultgames/pokerogue
# [Bug] Fix sketch being able to copy some unsketchable moves

## What are the changes?

Prevents Sketch from copying moves that are supposed to be [unsketchable](https://bulbapedia.bulbagarden.net/wiki/Sketch_(move))

## Why am I doing these changes?



To prevent these moves from being sketchable. 

## What did change?


With this change, you can no longer copy the following moves:
- Chatter
- Mirror Move
- Sleep Talk
- Struggle
- Sketch
- Revival Blessing
- Tera Starstorm
- Breakneck Blitz

NOTE: There are other moves that are technically unsketchable. I kept these as sketchable as other pokemon are given some of these moves as egg moves. These include:
- Dark Void 
- Hyperspace Fury 
- Wicked/Blazing/Noxious/Magical/Combat Torque 

### Screenshots/Videos


Before changes

https://github.com/user-attachments/assets/47d0bfe6-0763-4cfe-8b47-9dc7b51ce56a

After changes

https://github.com/user-attachments/assets/dda8dead-4886-4c9a-880f-6b5b30dea0fc

## How to test the changes?




Use Smeargle with Sketch. Do not give a mon Sketch with the overrides file as it does not properly replace the move. 
Give the opponent unsketchable moves, see that Sketch fails. 

## Checklist
